packages/wallets/src/components/Base/WalletPasswordField/WalletPasswordField.scss

```diff
@@ -0,0 +1,41 @@
+$meter-colors: (
+    initial: #e6e9e9,
+    weak: #ec3f3f,
+    moderate: #ffad3a,
+    strong: #4bb4b3,
+    complete: #4bb4b3,
+);
```

Comment:
mixins for themes (Dark and light)

Reply:
I replace these ones mentioned above with css variables however for implementing the whole colour palette should be in a separate PR/